Formula/elektra.rb

```diff
@@ -0,0 +1,42 @@
+class Elektra < Formula
+  desc "Configuration Framework"
```

Comment:
Can we get a slightly better description here? I have no idea what this means.

Reply:
I replaced the description with “Framework to access configuration settings in a global, hierarchical key database”. Hope that works for you.

Reply:
hopefully doesn't trigger the description-too-long audit :)